docs(google-maps): Update the workaround note to use the released version (#1712)
ionic-team/capacitor-plugins: google-maps/README.md

Read about [Configuring `Info.plist`](https://capacitorjs.com/docs/ios/configuration#configuring-infoplist) in the [iOS Guide](https://capacitorjs.com/docs/ios) for more information on setting iOS permissions in Xcode.

> > The main Google Maps SDK now supports running on simulators on Apple Silicon Macs, but make sure you are getting latest version of [Google-Maps-iOS-Utils](https://github.com/googlemaps/google-maps-ios-utils) by running `pod update Google-Maps-iOS-Utils` from `ios/App/` folder:

Reviewer: ```suggestion
> The main Google Maps SDK now supports running on simulators on Apple Silicon Macs, but make sure you have the latest version of [Google-Maps-iOS-Utils](https://github.com/googlemaps/google-maps-ios-utils) installed by running `pod update Google-Maps-iOS-Utils` from the `ios/App/` folder:
```

```
cd ios/App
pod update Google-Maps-iOS-Utils
```

> If you added the previous workaround you getting the unreleased version, you can delete it now by removing this line from `ios/App/Podfile`:

Reviewer: We should move this above running `pod update` since it should be done first or this one will just be re-installed.

Author: I've moved it up.
I didn't do it at first because new users will not have that workaround, so can be confusing for them, but if you think it could cause issues better put it first.
